src/asciidoc/docs/develop/testingTemplates.adoc

```diff
@@ -4,66 +4,59 @@
 
-=== Using live previews
+=== Using previews
 
 Your template might provide different experiences for different situations, or "contexts."
 For example, you might provide a different UI during a remix than when the template is running.
-The *Live preview* tab enables you to preview several standard contexts inside the Koji editor.
+The preview pane enables you to preview several standard contexts inside the editor.
 
-If a context is enabled in your template, a button for it appears in the *Live preview* pane.
-Click the button to display the preview.
+You can select from the following types of previews.
 
 * *Default* – Displays what a template looks like when running.
-This context is always enabled.
+The *Default* context is always enabled.
+
 * *Remix* – Displays what a template looks like in edit mode.
 A creator enters edit mode when they remix a template.
 The *Remix* context is always enabled, even if <<instant-remixing#,instant remixing>> is not.
-* *Thumbnail* – Displays what a template looks like when shared on social media.
-This context is always enabled.
-* *Admin*, *About*, and *Sticker* – Display previews of standard contexts, if you've enabled the associated entitlements.
+
+* *More* - Displays a menu of additional previews.
+The additional previews include the following.
+
+- *Admin*, *About*, and *Sticker* – Display previews of standard contexts, if you've enabled the associated entitlements.
 For more on contexts, see <<withkoji-vcc-package#_context>>.
-* *Buyer receipt* and *Seller receipt* – Displays previews of the transaction receipts for the buyer and seller.
-These contexts are available if you've implemented <<withkoji-koji-iap-package#_iap,in-app purchases>> and enabled the dynamic receipts entitlement.
 
-=== Using remote previews
+- *Thumbnail* – Displays what a template looks like when shared on social media.
+The *Thumbnail* context is always enabled.
 
-The *Remote* tab provides options for previewing a template on different devices or in different environments.
+- *Receipt (seller)* and *Receipt (buyer)* – Displays previews of the transaction receipts for the buyer and seller.
+These contexts are available if you've implemented <<withkoji-koji-iap-package#_iap,in-app purchases>> and enabled the dynamic receipts entitlement.
 
-* On a mobile device – Open a preview on any device by scanning a QR code.
-Logs from the device appear in a pane in the editor.
-* In a new tab – Open a preview on a staging server.
-Logs from the tab appear in a pane in the editor.
-* With a shared link – Copy the link for the preview.
-If you share this link with someone else, they can preview the template while it's still in development.
-+
-[NOTE]
-The remote preview link will only work for as long as your editor session is still active, so it is not suitable for long-term sharing.
+* *Remote previews* - The preview pane provides options for previewing a template on different devices or in different environments.
```

Comment:
Similar note as on the editor page. FYI, in the editor, all of the previews are running on a staging server (that's the dev environment for the editor).